src/data/artifact_inference.rs

```diff
@@ -0,0 +1,352 @@
+use camino::Utf8PathBuf;
+
+use super::artifacts::*;
+
+// Architectures
```

Comment:
are these commented out on purpose?

Reply:
Yeah they might be needed in the future, and wanted to leave them for reference for the js.